### ISSUE TITLE
Example site crashes on index GET request

### ISSUE THREAD

**Issue by 2xAA** [danmademe/express-vue-super-simple#1]
Possibly due to the mixin update recently?

Tried with lts/argon and lts/boron, same result.

```
/Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180
          throw new ReferenceError(messages.get("pluginUnknown", plugin, loc, i, dirname));
          ^

ReferenceError: Unknown plugin "add-module-exports" specified in "base" at 0, attempted to resolve relative to "/Users/twoxaa/projects/express-vue-super-simple"
    at /Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (native)
    at Function.normalisePlugins (/Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
    at OptionManager.mergeOptions (/Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:233:36)
    at OptionManager.init (/Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:367:12)
    at File.initOptions (/Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/node_modules/babel-core/lib/transformation/file/index.js:216:65)
    at new File (/Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/node_modules/babel-core/lib/transformation/file/index.js:139:24)
    at Pipeline.transform (/Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at scriptParser (/Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/dist/parser/index.js:84:45)
    at /Users/twoxaa/projects/express-vue-super-simple/node_modules/express-vue/dist/parser/index.js:99:26
```

**Comment by danmademe** [comment]
was missing `babel-plugin-add-module-exports`

i've updated the library

**Comment by 2xAA** [comment]
I cloned this without change

**Comment by danmademe** [comment]
ill do some further testing. but I'm guessing something didn't go right.

**Comment by 2xAA** [comment]
The same error occurs with express-vue-example by the way.

Just reinstalled the modules for this repo and ```babel-plugin-add-module-exports``` is included for sure:

![screen shot 2017-01-20 at 15 53 54](https://cloud.githubusercontent.com/assets/554219/22155812/ba8e3866-df28-11e6-8053-41c7794a8aa4.png)

The same error occurs

**Comment by danmademe** [comment]
which node version?

**Comment by 2xAA** [comment]
Installing with npm v3.10.10 and running with Node v 6.9.4 managed to solve this - sorry about that.
Must have installed with node v4 and the v3 npm twice over by accident.

Though, clicking one of the names in this example produces this error:
```(node:49627) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: Cannot read property 'mixins' of undefined```

**Comment by danmademe** [comment]
Ok I'm removing the dependency on this babel plugin.. ill push a new version soon..

Yeah i've seen the mixins issue, will solve this, but that issue is isolated to this repo..

express-vue@3.4.3 will be up as soon as unit tests complete, and i have dinner

**Comment by danmademe** [comment]
ok update to express-vue@3.4.3

**Comment by 2xAA** [comment]
Thanks - this is working fine with npm v3.10.10 and running with Node v6.9.4.

I know the mixins issue is different, but do you know why this is happening?
I'm trying to implement express-vue in my own express project and I get the same error.

I can open an issue on express-vue if that would be better.

**Comment by 2xAA** [comment]
Ah, it's just an easy 'check for undefined' fix.

I'll close this issue and let you fix the mixin issue separately - thanks for everything :)

**Comment by danmademe** [comment]
Just so you know I released 3.5.0 just now and added that null check.
